### PR TITLE
HRINT-3161 Pull latest docker image before build

### PR DESCRIPTION
**Pull request by hiddenshadow21**
### Issue link

https://issues.hibernatingrhinos.com/issue/HRINT-3161

### Additional description

Needs to be merged to v5.4 and v6.0 too.

### CODE REVIEW

File: scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile [hunk 1]
```diff
@@ -15,6 +15,8 @@ RUN apt update \
     && apt-get -y dist-upgrade \
     && apt install -y curl wget apt-transport-https 
 
+RUN for i in /etc/ssl/certs/*.pem; do HASH=$(openssl x509 -hash -noout -in $i); if [ ! -f /etc/ssl/certs/$HASH.0 ]; then ln -s $(basename $i) /etc/ssl/certs/$HASH.0; fi; done
```
Comment (gregolsky) [inline scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile:19]: interesting, why do we need this?

Comment (gregolsky) [reply scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile:19]: isnt this done with `sudo update-ca-certificates` or something?

Comment (gregolsky) [reply scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile:19]: or installing `ca-certificates` package?

Comment (hiddenshadow21) [reply scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile:19]: There is a problem with certificates while using `wget` on arm32v7 image on Ubuntu 20.04.
```
Resolving packages.microsoft.com (packages.microsoft.com)... 13.90.56.68
Connecting to packages.microsoft.com (packages.microsoft.com)|13.90.56.68|:443... connected.
ERROR: cannot verify packages.microsoft.com's certificate, issued by 'CN=Microsoft Azure TLS Issuing CA 05,O=Microsoft Corporation,C=US':
  Self-signed certificate encountered.
To connect to packages.microsoft.com insecurely, use `--no-check-certificate'.
```

Unfortunately `update-ca-certificates` does not solve this.

Comment (gregolsky) [reply scripts/linux/pkg/deb/ubuntu_multiarch.Dockerfile:19]: that's a workaround, we'll pass a comment with a SO link

**Review by gregolsky — APPROVED** [review]

**Pull request merged**